3-Authorization-II/1-call-api/SPA/src/app/auth-config.ts

```diff
@@ -10,21 +10,26 @@ import { LogLevel, Configuration, BrowserCacheLocation } from '@azure/msal-brows
 const isIE = window.navigator.userAgent.indexOf("MSIE ") > -1 || window.navigator.userAgent.indexOf("Trident/") > -1;
 
 /**
- * Configuration object to be passed to MSAL instance on creation. 
+ * Configuration object to be passed to MSAL instance on creation.
  * For a full list of MSAL.js configuration parameters, visit:
- * https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/configuration.md 
+ * https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/configuration.md
  */
 export const msalConfig: Configuration = {
   auth: {
     clientId: 'Enter_the_Application_Id_Here', // This is the ONLY mandatory field that you need to supply.
     authority: 'https://login.microsoftonline.com/Enter_the_Tenant_Info_Here', // Defaults to "https://login.microsoftonline.com/common"
     redirectUri: '/', // Points to window.location.origin. You must register this URI on Azure portal/App Registration.
```

Comment:
You might want to add a redirect page to follow the MSAL best practices

Reply:
done